modules/openapi-generator/src/test/java/org/openapitools/codegen/swift4/Swift4ModelEnumTest.java

```diff
@@ -51,8 +54,80 @@ public void converterTest() {
         Assert.assertEquals(enumVar.dataType, "String");
         Assert.assertEquals(enumVar.datatypeWithEnum, "Name");
         Assert.assertEquals(enumVar.name, "name");
-        Assert.assertEquals(enumVar.defaultValue, ".VALUE2");
+        Assert.assertEquals(enumVar.defaultValue, ".value2");
```

Comment:
With enum, name of default value should be converted as enum var name.

In current master version generator, if you use following declaration...

``` yaml
openapi: 3.0.0
info:
  description: ''
  version: 1.0.0
  title: ''
paths:
  /items:
    post:
      description: ''
      operationId: addItem
      requestBody:
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/NewItem'
      responses:
        '405':
          description: Invalid input
components:
  schemas:
    NewItem:
      properties:
        size:
          type: string
          enum:
          - VALUE1
          - VALUE2
          - VALUE3
          default: VALUE2
```

... following code will be generated.

``` swift
import Foundation

public struct NewItem: Codable {

    public enum Size: String, Codable {
        case value1 = "VALUE1"
        case value2 = "VALUE2"
        case value3 = "VALUE3"
    }
    public var size: Size? = .VALUE2

    public init(size: Size?) {
        self.size = size
    }

}
```

This code fails to compile.

This is because, enum var name is camelized at following.

> https://github.com/OpenAPITools/openapi-generator/pull/4481/files#diff-01cf6e8770037e6a2a690476f2310d57R776-R781

So name of default value for enum should be converted as enum var name by `toEnumVarName`, even if the type is string.

After apply this PR's fix, code will be generated as follows.

``` swift
import Foundation

public struct NewItem: Codable {

    public enum Size: String, Codable {
        case value1 = "VALUE1"
        case value2 = "VALUE2"
        case value3 = "VALUE3"
    }
    public var size: Size? = .value2

    public init(size: Size?) {
        self.size = size
    }

}
```